contrib/exporters/secadvisor/mod/resolve_runc.go

```diff
@@ -44,6 +45,9 @@ func (r *resolveRunc) IPToName(ipString, nodeTID string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if (node == nil) {
```

Comment:
redundant as gremlinClinet.GetNode() already addresses this case

Reply:
OK